CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## UNRELEASED
+
+* Converted library and Demo project to Swift 3.  
```

Comment:
Hey @ceyhuno, you forgot to credit yourself! (I think the other people who worked on the Swift 3 migration like @phatblat on #30 deserve credits there too, btw)

Could you add there the links to your profiles and to the PRs related to that migration? (you can use other entries in the CHANGELOG as example)